src/api/notify-namespace.ts

```diff
@@ -635,25 +635,8 @@ export class NotifyNamespace {
  *
  * @internal
  */
-const WEBHOOK_NETWORK_TO_NETWORK: { [key: string]: Network } = {
-  ETH_MAINNET: Network.ETH_MAINNET,
-  ETH_GOERLI: Network.ETH_GOERLI,
-  ETH_SEPOLIA: Network.ETH_SEPOLIA,
-  MATIC_MAINNET: Network.MATIC_MAINNET,
-  MATIC_MUMBAI: Network.MATIC_MUMBAI,
-  MATIC_AMOY: Network.MATIC_AMOY,
-  ARB_MAINNET: Network.ARB_MAINNET,
-  ARB_GOERLI: Network.ARB_GOERLI,
-  ARB_SEPOLIA: Network.ARB_SEPOLIA,
-  OPT_MAINNET: Network.OPT_MAINNET,
-  OPT_GOERLI: Network.OPT_GOERLI,
-  OPT_SEPOLIA: Network.OPT_SEPOLIA,
-  BASE_MAINNET: Network.BASE_MAINNET,
-  BASE_GOERLI: Network.BASE_GOERLI,
-  BASE_SEPOLIA: Network.BASE_SEPOLIA,
-  ZKSYNC_MAINNET: Network.ZKSYNC_MAINNET,
-  ZKSYNC_SEPOLIA: Network.ZKSYNC_SEPOLIA
-};
+const WEBHOOK_NETWORK_TO_NETWORK: { [key: string]: Network } =
+  Object.fromEntries(Object.entries(Network));
```

Comment:
Why does this work? Were we not including all the networks in this list?

Reply:
Yes, it generates the same map from the Network enum on the fly. 
This will ensure that we do not forget to add networks here in the future.

We could probably refactor this to remove these maps altogether and use the Network enum directly, but wanted to keep the changes to the minimum.

![Screenshot 2024-10-22 at 00 56 59](https://github.com/user-attachments/assets/d9e2e779-bb48-4250-a4a9-156ed0627d6e)

Reply:
@thebrianchen the only thing that this object does to prevent that is replace the network field in the request with `undefined` before making webhook requests. 

It's not like it's doing any validation before making the requests.

Why not just pass on the unsupported network to the webhooks service and let it handle the error instead of nullifying the network field?

Reply:
I think they wanted to release gate some of the webhooks, and what i had here was definitely not the best way to do it.